.github/workflows/build-and-push.yml

```diff
@@ -35,41 +42,65 @@ jobs:
           ver="${{ matrix.version }}"
           echo "::set-output name=SHORT_VER::${ver//./}"
 
-      - name: Check if dockerfile is present in version directory
-        id: check_dockerfile_file
+      - name: Login to Quay.io private registry
```

Comment:
Shouldn't this step be just before the push-to-quay step, and not here? 
That way, when the build fails, the quay.io would not be to be contacted at all.

Reply:
Good, point. I wanted to have at the beginning. When the login is called, then it is valid for action.

I wanted to have a setup for the action `build-and-push` at the beginning.

Reply:
It's true, that from the past can be seen, that login to quay.io is more error-prone than building the containers.